tensorflow/python/keras/losses_test.py

```diff
@@ -632,6 +632,15 @@ def test_zero_weighted(self):
     loss = msle_obj(y_true, y_pred, sample_weight=0)
     self.assertAlmostEqual(self.evaluate(loss), 0.0, 3)
 
+  def test_ragged_tensors(self):
+    msle_obj = losses.MeanSquaredLogarithmicError()
+    y_true = ragged_factory_ops.constant([[1, 9, 2], [-5, -2]])
+    y_pred = ragged_factory_ops.constant([[4, 8, 12], [8, 1]],
+                                         dtype=dtypes.float32)
+    sample_weight = constant_op.constant([1.2, 3.4], shape=(2, 1))
+    loss = msle_obj(y_true, y_pred, sample_weight=sample_weight)
+    self.assertAlmostEqual(self.evaluate(loss), 5.1121, 3)
```

Comment:
Can you add some comments to this test noting the actual computation you expect to take place here and why it should come out to 5.11? (e.g. how you did in one of the other losses

Reply:
Done.

Reply:
You might need to push again?

Reply:
My bad. Sorry.